app/process/watcher/consensus-message/consensus-message.go

```diff
@@ -88,15 +90,29 @@ func (ctw ConsensusTopicWatcher) subscribeToTopic(q *queue.Queue) {
 	log.Infof("Started Consensus Message Watcher for topic [%s]\n", ctw.topicID)
 	unprocessedMessages, err := ctw.client.GetUnprocessedMessagesAfterTimestamp(ctw.topicID, milestoneTimestamp)
 	if err != nil {
+		log.Errorf("Could not get unprocessed messages after timestamp [%s]", milestoneTimestamp)
 		log.Fatal(err)
 	}
 	ctw.started = true
+	log.Println(unprocessedMessages)
 	for _, u := range unprocessedMessages.Messages {
-		// validate and check body
-		msg := &protomsg.TopicSignatureMessage{}
+		decodedMessage, err := b64.StdEncoding.DecodeString(u.Message)
+		if err != nil {
+			log.Errorf("Could not decode message - [%s]", u.Message)
```

Comment:
I think that we should add info that we are going to skip the processing of this message:
"Could not decode message - [%s]. Skipping the processing of this message", u.Message